src/extensions/email_provider_rate_limit_extension.ts

```diff
@@ -0,0 +1,22 @@
+import { flattenErrors, statusCodeForError } from "lib/graphqlErrorHandler"
```

Comment:
Minor, but can we rename this file to `emailProviderRateLimitExtension` so that it matches idiomatic JS? We have snake_case in MP, but that's very old.